content/en/cloud_cost_management/aws.md

```diff
@@ -19,11 +19,11 @@ further_reading:
 
 ## Overview
 
-To use AWS Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a Cost and Usage report.
+To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a legacy Cost and Usage report.
```

Comment:
```suggestion
To use Cloud Cost Management, you must have an AWS account with access to Cost and Usage Reports (CURs), and have the AWS integration installed in Datadog. To set up Cloud Cost Management in Datadog, you need to generate a legacy Cost and Usage report through AWS Billing.
```

Reply:
Targeting this: 
<img width="359" alt="image" src="https://github.com/DataDog/documentation/assets/17853509/3d52e962-0680-4858-bab2-16f91a55726e">

and not this:
<img width="1542" alt="image" src="https://github.com/DataDog/documentation/assets/17853509/24fc95dd-f8ea-4bf8-ae70-417e6c6f41be">